Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml

```diff
@@ -264,6 +268,11 @@
   parent: [BaseBorgChassis, BaseBorgTransponder]
   abstract: true
   components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyScience
+      - EncryptionKeyCommon
```

Comment:
In your PR description you said they only start with common and binary access, but here they seem to have a science key in their key_slot container?